docs/docs/rest-api/public/api/v2/tasks.raml

```diff
@@ -0,0 +1,43 @@
+get:
+  description:
+    List all running tasks.
+  is: [ secured ]
+  queryParameters:
+    status:
+      description: Filter the list of tasks by status
+      enum: [ TASK_STAGING, TASK_STARTING, TASK_RUNNING, TASK_FINISHED, TASK_FAILED, TASK_KILLED, TASK_LOST, TASK_ERROR ]
```

Comment:
1) the implementation will match the given string like this
```
    case "running" => Some(TaskState.TASK_RUNNING)
    case "staging" => Some(TaskState.TASK_STAGING)
    case _         => None
```
so passing `TASK_STAGING ` will **not** work but return all tasks disregarding the status. In that sense, the documentation is still wrong.

2) Yes, the base for the result of this endpoint is the TaskTracker list, so integrating any other task states would be quite easy.

Please fix 1)